### PR TITLE
Address apiview comments for CosmosDB

### PR DESCRIPTION
**Pull request by ArthurMa1978**

### CODE REVIEW

File: sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/Generated/Models/CassandraCommandPostBody.cs [hunk 1]
```diff
@@ -43,6 +43,6 @@ public CassandraCommandPostBody(string command, string host)
         /// <summary> If true, stops cassandra before executing the command and then start it again. </summary>
         public bool? CassandraStopStart { get; set; }
         /// <summary> If true, allows the command to *write* to the cassandra directory, otherwise read-only. </summary>
-        public bool? Readwrite { get; set; }
+        public bool? IsReadwrite { get; set; }
```
Comment (fengzhou-msft) [inline sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/Generated/Models/CassandraCommandPostBody.cs:46]: Based on the description, `AllowReadWrite` may be a better name?

Comment (ArthurMa1978) [reply sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/Generated/Models/CassandraCommandPostBody.cs]: Good suggestion, renamed to `AllowWrite`

**Review by ArcturusZhang — APPROVED** [review]
It would be nice if we could move those `rename-model` into `rename-mapping`

File: sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/Generated/Models/CosmosDBServerVersion.cs [hunk 2]
```diff
@@ -22,16 +22,10 @@ public CosmosDBServerVersion(string value)
...
-        /// <summary> 3.2. </summary>
-        public static CosmosDBServerVersion Three2 { get; } = new CosmosDBServerVersion(Three2Value);
-        /// <summary> 3.6. </summary>
-        public static CosmosDBServerVersion Three6 { get; } = new CosmosDBServerVersion(Three6Value);
-        /// <summary> 4.0. </summary>
-        public static CosmosDBServerVersion Four0 { get; } = new CosmosDBServerVersion(Four0Value);
+        private const string V3_2Value = "3.2";
+        private const string V3_6Value = "3.6";
+        private const string V4_0Value = "4.0";
+        private const string V4_2Value = "4.2";
```
Comment (Yao725) [inline sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/Generated/Models/CosmosDBServerVersion.cs:28]: Interesting, why we don't have `4.2` in the previous generated code?

**Review by Yao725 — APPROVED** [review]

File: sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/Generated/Models/CosmosDBServerVersion.cs (hunk 2 above)
Comment (ArcturusZhang) [reply sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/Generated/Models/CosmosDBServerVersion.cs:28]: Arthur updated the swagger commit

**Review by fengzhou-msft — APPROVED** [review]

File: sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/Generated/Models/CosmosDBServerVersion.cs (hunk 2 above)
Comment (ArthurMa1978) [reply sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/Generated/Models/CosmosDBServerVersion.cs:28]: That's the problem that we not use `readme.md` under spec repo

**Pull request merged**